build_tools/benchmarks/README.md

```diff
@@ -0,0 +1,49 @@
+# IREE Benchmark Suites Tool
+
+This directory contains the tools to run IREE benchmark suites and generate
+reports. More information about benchmark suites can be found [here](benchmarks/README.md).
+
+## Benchmark Tools
+Currently we have `run_benchmarks_on_android.py` and `run_benchmarks_on_linux.py` to run
+benchmark suites on Android devices (with `adb`) and Linux local machines.
+
+The available arguments can be shown with `--help`. Here we list some common
+usages (assume the target `iree-benchmark-suites` is built under `$IREE_BUILD`):
+
+**Run all benchmarks**
+```sh
+./run_benchmarks_on_linux.py \
+  --normal_benchmark_tool_dir=$IREE_BUILD/iree/tools \
+  --output=results.json $IREE_BUILD
+```
+
+**Run all benchmarks and capture the tracy files**
+```sh
+./run_benchmarks_on_linux.py \
+  --normal_benchmark_tool_dir=$IREE_BUILD/iree/tools \
+  --traced_benchmark_tool_dir=$IREE_TRACED_BUILD/iree/tools \
+  --trace_capture_tool=${path of iree-tracy-capture} \
+  --capture_tarball=captured_tracy_files.tar
+  --output=results.json $IREE_BUILD
```

Comment:
`$IREE_BENCHAMRK_SUITE_BUILD_DIR`?

It would be helpful to explain what each of these mean too.

Reply:
I explained the IREE_BUILD_DIR in the section above.